ui/page/start_page.go

```diff
@@ -23,6 +24,13 @@ type (
 	D = layout.Dimensions
 )
 
+type onBoardingScreen struct {
+	onBoardingScreenTitle string
```

Comment:
```suggestion
	Title string
```

just title is sufficient

`onBoardingScreen.Title`